gulp-core-build-karma/src/KarmaTask.ts

```diff
@@ -67,6 +78,30 @@ export class KarmaTask extends GulpTask<IKarmaTaskConfig> {
 
       completeCallback();
     } else {
+      // Normalize the match expression if one was specified
+      const { testMatch } = this.taskConfig; // tslint:disable-line:typedef
+      if (testMatch) {
+        let normalizedMatch: RegExp;
+        if (typeof testMatch === 'string') {
+          normalizedMatch = new RegExp(testMatch as string);
+        } else if (testMatch instanceof RegExp) {
+          normalizedMatch = testMatch;
+        } else {
+          completeCallback('The testMatch regular expression is invalid');
```

Comment:
The RegExp constructor can throw if you pass it an invalid pattern, so we should probably also have a catch.